src/test/java/io/github/opencubicchunks/cubicchunks/test/misc/TestInterfacesMatchVanillaClasses.java

```diff
@@ -97,7 +95,8 @@ private void testParityIncludingAncestors(Class<?> vanillaClass, Class<?> cubicC
             LevelChunk.class.getMethod("setData", AttachmentType.class, Object.class),
             LevelChunk.class.getMethod("hasData", Supplier.class),
             LevelChunk.class.getMethod("getData", Supplier.class),
-            LevelChunk.class.getMethod("setData", Supplier.class, Object.class)
+            LevelChunk.class.getMethod("setData", Supplier.class, Object.class),
+            LevelChunk.class.getMethod("getAuxLightManager", ChunkPos.class)
```

Comment:
Why did this not cause a failure in main ci?

Reply:
Bumped neoforge version in this PR and it apparently added this method